Header header
# The position of the virtual leader
geometry_msgs/Point position
# The velocity of the virtual leader
geometry_msgs/Vector3 velocity
